### PR TITLE
Use SLURM rules for export.{csv/json}

### PR DESCRIPTION
**Pull request by lolepezy**
Bug reported here
https://github.com/RIPE-NCC/rpki-validator-3/issues/185

### CODE REVIEW

File: rpki-validator/src/main/java/net/ripe/rpki/validator3/api/roas/ExportsController.java [hunk 1]
```diff
@@ -123,74 +121,114 @@ public void exportCsv(HttpServletResponse response) throws IOException {
...
     }
 
+    /**
+     * Get the stream of VRPs based on ROAs - whitelist + SLURM assertions.
+     */
+    private Stream<VRP> vrpStream() {
+        Stream<VRP> validatedPrefixes = validatedRpkiObjects
+            .findCurrentlyValidatedRoaPrefixes()
+            .getObjects()
+            .filter(new IgnoreFiltersPredicate(ignoreFilters.all()).negate())
```
Comment (wibisono) [inline rpki-validator/src/main/java/net/ripe/rpki/validator3/api/roas/ExportsController.java:198]: Somehow we need to mark PR with refactoring where to focus on most relevant changes :P 

Here is what changed and makes difference on the whole story :)

File: rpki-validator/src/main/java/net/ripe/rpki/validator3/api/roas/ExportsController.java [hunk 2]
```diff
@@ -123,74 +121,114 @@ public void exportCsv(HttpServletResponse response) throws IOException {
...
+        final Stream<VRP> assertions = roaPrefixAssertions
+            .all()
+            .map(assertion -> new VRP(
+                assertion.getAsn().toString(),
+                assertion.getPrefix().toString(),
+                assertion.getMaxPrefixLength() != null ? assertion.getMaxPrefixLength() : assertion.getPrefix().getPrefixLength(),
+                "SLURM"
+            ));
+
+        return Stream.concat(validatedPrefixes, assertions).distinct();
```
Comment (ties) [inline rpki-validator/src/main/java/net/ripe/rpki/validator3/api/roas/ExportsController.java:215]: VRPs with an differing `ta` will not be seen as distinct (so SLURM versus from any of the trust anchors will both be there)

**Comment by ties** [comment]
Rebased: Should remove the failing checks since they are no longer in master.

File: rpki-validator/src/main/java/net/ripe/rpki/validator3/api/roas/ExportsController.java (hunk 1 above)
Comment (lolepezy) [reply rpki-validator/src/main/java/net/ripe/rpki/validator3/api/roas/ExportsController.java:198]: Yeah :)
Just had a fit of boyscouting.

File: rpki-validator/src/main/java/net/ripe/rpki/validator3/api/roas/ExportsController.java (hunk 2 above)
Comment (lolepezy) [reply rpki-validator/src/main/java/net/ripe/rpki/validator3/api/roas/ExportsController.java:215]: I guess the point was to just return all VRPs regardless of their priority or origin, so conflict resolution and prioritising is up to the consumer.

**Comment by lolepezy** [comment]
Fixed in the version 3.1-2020.05.08.09.26.49 that we just released.

**Pull request merged**